testing/endtoend/mainnet_scenario_e2e_test.go

```diff
@@ -1,18 +1,21 @@
 package endtoend
 
 import (
-	"testing"
-
 	"github.com/prysmaticlabs/prysm/v3/testing/endtoend/types"
+	"testing"
 )
 
 func TestEndToEnd_MainnetConfig_MultiClient(t *testing.T) {
 	e2eMainnet(t, false /*usePrysmSh*/, true /*useMultiClient*/).run()
 }
 
-func TestEndToEnd_MultiScenarioRun_Multiclient(t *testing.T) {
-	runner := e2eMainnet(t, false /*usePrysmSh*/, true /*useMultiClient*/, types.WithEpochs(22))
-	runner.config.Evaluators = scenarioEvalsMulti()
-	runner.config.EvalInterceptor = runner.multiScenarioMulticlient
-	runner.scenarioRunner()
+func TestEndToEnd_MainnetConfig_Multiclient_CrossClient(t *testing.T) {
+	e2eMainnet(t, false /*usePrysmSh*/, true /*useMultiClient*/, types.WithValidatorCrossClient()).run()
 }
+
+//func TestEndToEnd_MultiScenarioRun_Multiclient(t *testing.T) {
+//	runner := e2eMainnet(t, false /*usePrysmSh*/, true /*useMultiClient*/, types.WithEpochs(22))
+//	runner.config.Evaluators = scenarioEvalsMulti()
+//	runner.config.EvalInterceptor = runner.multiScenarioMulticlient
+//	runner.scenarioRunner()
+//}
```

Comment:
Commented code

Reply:
I was still testing lol will be uncommented soon